RevenueCatUI/PaywallView.swift

```diff
@@ -321,14 +321,20 @@ struct LoadedOfferingPaywallView: View {
 
     @ViewBuilder
     private var content: some View {
+        let configuration = self.paywall.configuration(
```

Comment:
Nitpick, but I wonder if we should move creating the configuration outside the view... Having said that, it would need all these parameters, so I think it's ok as is, at least for now.